addons/io_sketchfab_plugin/blender/imp/gltf2_blender_map_emissive.py

```diff
@@ -101,6 +102,16 @@ def create_cycles(gltf, material_idx):
             node_tree.links.new(combine.inputs[0], math_R.outputs[0])
             node_tree.links.new(combine.inputs[1], math_G.outputs[0])
             node_tree.links.new(combine.inputs[2], math_B.outputs[0])
-            node_tree.links.new(principled.inputs[17], combine.outputs[0])
+            #node_tree.links.new(principled.inputs[17], combine.outputs[0])
+        else:
+            pass#node_tree.links.new(principled.inputs[17], text.outputs[0])
+
```

Comment:
We didn't discussed this yet but it would be nice to be PEP-8 compliant (using flake8)